cryptoinvestor/models/Account.py

```diff
@@ -0,0 +1,4 @@
+class Account:
+
+    def __init__(self, *, balance=0):
```

Comment:
Using `asterisk` in method definition says that all arguments after are taken as positional and do not have defaults. Since `balance` has a default, the `asterisk` is irrelevant here.